doc/api.rst

```diff
@@ -114,9 +114,12 @@ string checks, wide string, output iterator and user-defined type support.
 Compile-time Format String Checks
 ---------------------------------
 
-Compile-time checks are supported for built-in and string types as well as
-user-defined types with ``constexpr`` ``parse`` functions in their ``formatter``
-specializations.
+Compile-time checks are enabled when using ``FMT_STRING``. It supports built-in
+and string types as well as user-defined types with ``constexpr`` ``parse``
+functions in their ``formatter`` specializations.
+
+When the compiler supports UDL templates extensions, using the litteral string
+``""_format()`` will also enable compile-time checks.
```

Comment:
I suggest dropping this part because the extension is obsolete (gcc even warns on it now). `FMT_STRING` is the only recommended API for compile-time checks.